recipe/meta.yaml

```diff
@@ -47,6 +47,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
+        - {{ pin_subpackage('ray-default', exact=True) }}
```

Comment:
shouldn't pinning `ray-default` be enough? I presume that it depends on `ray-core` by itself... and that builds a little easier to follow dependency chain IMHO

Reply:
Fine by me either way.